src/assets/styles/mixins/loader.scss

```diff
@@ -3,7 +3,7 @@
 @mixin background-logo-loader($zoom: 1) {
   &::after {
     animation: loading 2s linear infinite;
-    background: transparent url('~assets/customicons/truenas_scale_logo_full.svg') no-repeat center center;
+    background: transparent url('/src/assets/customicons/truenas_scale_logo_full.svg') no-repeat center center;
```

Comment:
There is regression on main dashboard and reporting page where loading state does not show truenas logo.